azure-pipelines.yml

```diff
@@ -98,8 +96,6 @@ stages:
         TOPOLOGY: t1-lag
 
   - job:
-    pool:
-      vmImage: 'ubuntu-20.04'
```

Comment:
@wangxin  I remember if remove ubuntu-20.04 and choose default, the default host os version is ubuntu 16, the pipeline will fail. Not sure if it's the reason to define specific vmImage.

Reply:
The default image used by Azure Pipeline is configurable. Previously the pipeline was configured to use "Ubuntu 16" which is not available. The pipeline has been configured to use default "Azure Pipeline" agent pool which is always available. Currently the image of "Azure Pipeline" agent pool is ubuntu 20.04 if I remember correctly.